lib/facter/util/ciscoos.rb

```diff
@@ -0,0 +1,22 @@
+module Facter
+  module Util
+    module CiscoOS 
+
+      # @file follows the same format as os-release
+      # @see http://www.freedesktop.org/software/systemd/man/os-release.html
+      def self.cisco_release(file)
```

Comment:
Do we need a cisco-specific implementation for reading a release file?  We read release files for other operating systems, so is there something specific to this implementation that we need for this to work?